pkg/repositories/config/migrations.go

```diff
@@ -359,6 +359,17 @@ var Migrations = []*gormigrate.Migration{
 			return alterTableColumnType(db, "id", "int")
 		},
 	},
+
+	// Add state to execution model.
+	{
+		ID: "2022-01-11-execution-state",
+		Migrate: func(tx *gorm.DB) error {
+			return tx.AutoMigrate(&models.Execution{})
+		},
+		Rollback: func(tx *gorm.DB) error {
+			return tx.Table("execution").Migrator().DropColumn(&models.Execution{}, "state")
```

Comment:
and "state_updated_by" too?

Reply:
removed stateUpdatedBy column as we are tracking it as part of the closure